lib/fog/openstack/identity.rb

```diff
@@ -35,14 +39,6 @@ def initialize(options = {})
           @openstack_identity_prefix = options[:openstack_identity_prefix]
```

Comment:
`options[:openstack_identity_prefix]` is used widely, but not `@openstack_identity_prefix` - that is only used in the Mock class in this file (and is there only for the v3? function) and is completely unused as a result of this commit.